_topics/en/version-3-transaction-relay.md

```diff
@@ -46,20 +53,19 @@ see_also:
 
   - title: Anchor outputs
     link: topic anchor outputs
+---
+V3 transaction relay is a superset of standard transaction policy.
+That is, V3 transactions follow all rules for standard transactions
+(e.g. minimum and max transaction weight) while also adding some
+non-standard rules designed to allow transaction replacement
+while precluding transaction-pinning attacks. V3 transactions also
+require minor changes to the Package RBF policy in order to maintain
```

Comment:
Package RBF can link to `[topic package relay]`.

Reply:
done :)